package.json

```diff
@@ -1,14 +1,20 @@
 {
-  "name": "cast-monorepo",
+  "name": "cast",
   "version": "0.1.0",
-  "description": "A monorepo for the Cast app",
-  "private": true,
+  "description": "A podcast listening PWA",
   "scripts": {
-    "preinstall": "npx only-allow pnpm",
```

Comment:
[nitpick (non-blocking)](https://conventionalcomments.org/#labels): this was also nice to have, let's bring it back